src/main/java/frc/robot/Robot.java

```diff
@@ -1,25 +1,12 @@
-/*----------------------------------------------------------------------------*/
```

Comment:
I think we want to leave this header.

Reply:
Really? Why? It is copyright by LigerBots, not WPILib.